app.json

```diff
@@ -18,5 +18,12 @@
   },
   "ios": {
     "bundleIdentifier": "com.shapeShift.shapeShift"
+  },
+  "build": {
+    "production": {
+      "ios": {
+        "image": "latest"
+      }
+    }
```

Comment:
Do we need this @NeOMakinG? `expo doctor` isn't happy with it:
```shell
Error: Problem validating fields in app.json. Learn more: https://docs.expo.dev/workflow/configuration/
 • should NOT have additional property 'build'.
```

Reply:
No I think it's not needed, we can remove it